libvast/src/table_slice.cpp

```diff
@@ -49,7 +49,9 @@ auto visit(Visitor&& visitor, const fbs::TableSlice* x) noexcept(
     // Check whether the handlers for all other table slice encodings are
     // noexcept-specified. When adding a new encoding, add it here as well.
     std::is_nothrow_invocable<Visitor>,
+#if VAST_HAVE_ARROW
```

Comment:
Please add a comment for why this is needed here, as you did in the PR description.

Otherwise green light if CI checks out.

Reply:
Done.